ws-client/ws.q

```diff
@@ -21,4 +21,12 @@ open0:{[x;y;v]
 
 open:{neg first open0[x;y;.ws.VERBOSE]}                                 //return neg handle for messaging
 
+.ws.close:{[h]
+  if[all(h in key .ws.w;h in key .z.W);hclose h];                       //close handle if h is found both in .ws.w and .z.W (all opened handles)
+  .ws.w:.ws.w _ h;                                                      //remove h from .ws.w
+  .ws.servers: .ws.servers _ h;                                         //remove h from .ws.servers
```

Comment:
think we could just call `.z.wc` here which is already defined to remove from servers/clients tables

```suggestion
  .z.wc h;                                         //remove h from .ws.servers
```